.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "app/console"]
 	path = app/console
 	url = https://github.com/appwrite/console
-	branch = 2.3.4
```

Comment:
We should probably still use the latest here to avoid too many conflicts

Reply:
We have a new branch for 1.4 . Once we start merging more PRs, itll be easier to change this